SDK_versioned_docs/version-3.0.0/widgets/swap-widget-api.md

```diff
@@ -10,29 +10,29 @@ sidebar_position: 3
+| `defaultOutputTokenAddress` | `{[chainId: number]: string}`                                   | `string` or `undefined` | Address of the token to be selected by default in the input field (e.g. USDC) for each network chain ID. None if left empty. Any addresses provided in this parameter must be included in the `tokenList`.                                                                                                                                                                                                                                                                                                                           |
+| `defaultOutputTokenAmount`  | `number`                                                        | `0`                     | Default amount for the input field (e.g. 100 USDC). This value will respect the decimals of the `defaultOutputTokenAddress`. This parameter is mutually exclusive with `defaultInputTokenAmount`, so you may set only one `of defaultInputTokenAmount and` `defaultOutputTokenAmount`.                                                                                                                                                                                                                                               |
+| `locale`                    | `SupportedLocale`                                               | `en-US`                 | Specifies an explicit locale to use for the widget interface. This can be set to one of the values exported by the library in [`SUPPORTED_LOCALES`](https://github.com/Uniswap/widgets/blob/main/src/constants/locales.ts).                                                                                                                                                                                                                                                                                                          |
+| `onClickConnectWallet`      | `(e?: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void` | `undefined`             | This callback is triggered when the 'Connect your wallet' message in the widget is clicked, and can be used to trigger your own wallet connection flow from the widget. Disable the built-in wallet connection flow by calling `e?.preventDefault()`.                                                                                                                                                                                                                                                                                |
```

Comment:
is the built in flow also disabled if the integrator leaves this undefined?

Reply:
nope - built-in flow will exist by default unless integrator calls `e?.preventDefault()`. cc @zzmp's rec

Reply:
thought through this with @akarys92  - going to just leave it to e.preventDefault

`onClickConnectWallet?: false | ((e?: React.MouseEvent<HTMLButtonElement>) => void)` introduces confusion & isn't a huge quality of life improvement anyway